frontend/src/static/js/v_authorship.js

```diff
@@ -294,6 +312,10 @@ window.vAuthorship = {
           });
       return numLinesModified;
     },
+
+    numberOfBinaryFiles() {
```

Comment:
`getBinaryFilesCount` will probably be better

Reply:
I don't think the `get` is necessary since it is a computed value. How about just `binaryFilesCount`?

Reply:
yes, that would be nice; keep it consistent to the naming convention

Reply:
Will do.